.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish Package
+
+on:
+  release:
+    types: [published]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+        registry-url: 'https://registry.npmjs.org'
+    - name: Install Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install packaging setuptools twine wheel
+    - name: Publish the Python package
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
How are we supposed to generate and set this token?

Reply:
For generating go to pypi-> account settings then scroll down to:
![image](https://user-images.githubusercontent.com/10111092/103562112-88a58600-4e88-11eb-9f53-f013a51b55da.png)

For setting you add it to the `Secrets` section of this repo in settings like so:
![image](https://user-images.githubusercontent.com/10111092/103562164-9ce98300-4e88-11eb-9c5c-c7957afbf179.png)


then it all sorta "just works"